### PR TITLE
LoggerStructureFormatter -> Internal Namespace

### PR DESCRIPTION
**Pull request by BrennanConroy**
#123

### CODE REVIEW

File: src/Microsoft.Framework.Logging.Interfaces/project.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
         "aspnet50": { },
         "aspnetcore50": {
             "dependencies": {
+                "System.Collections": "4.0.10-*",
```
Comment (davidfowl) [inline src/Microsoft.Framework.Logging.Interfaces/project.json]: The other versions have beta-*

**Comment by Eilon** [comment]
Move the file too. It's in the wrong folder.

**Comment by BrennanConroy** [comment]
Shouldn't it be in the Microsoft.Framework.Logging folder?

**Comment by Eilon** [comment]
Namespaces and folder names are supposed to match.

**Comment by BrennanConroy** [comment]
There isn't a .Internal folder, I can make one if we want to change that

**Comment by Eilon** [comment]
`md Internal` :smile:

**Comment by BrennanConroy** [comment]
Should we make a new project for it?

**Comment by Eilon** [comment]
For interfaces that used to be ANI's we create new projects. But this is just some random type we want to stuff into an `Internal` namespace because it's an implementation detail, no?

**Comment by BrennanConroy** [comment]
I looked at some other repos and saw how they did it, so I copied them. Just put a Internal folder inside the project

**Comment by Eilon** [comment]
Yup.

**Pull request closed without merging**